mzLib/Readers/ExternalResults/IndividualResultRecords/ExperimentAnnotation.cs

```diff
@@ -0,0 +1,45 @@
+﻿using CsvHelper.Configuration;
+using CsvHelper.Configuration.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Readers
+{
+    /// <summary>
+    /// A class representing a single entry in an experiment_annotation.tsv file
+    /// </summary>
+    public class ExperimentAnnotation
+    {
+        public static CsvConfiguration CsvConfiguration = new CsvConfiguration(CultureInfo.InvariantCulture)
+        {
+            Delimiter = "\t",
+            HasHeaderRecord = true,
+            IgnoreBlankLines = true,
+            TrimOptions = TrimOptions.Trim,
+            BadDataFound = null,
```

Comment:
BadDataFound = null will skip all lines it cannot understand, instead of crashing. Is this the intended behavior?